R/mcmc-traces.R

```diff
@@ -421,6 +435,111 @@ mcmc_rank_hist <- function(x,
     labs(x = "Rank")
 }
 
+#' @rdname MCMC-traces
```

Comment:
Can you double check if all arguments are documented? For example, I see no documentation of `interpolate_adj`.